clusterloader2/pkg/prometheus/manifests/prometheus-windows-scrape-configs.yaml

```diff
@@ -8,5 +8,7 @@ metadata:
 stringData:
   windows-scrape-configs.yaml: |-
     - job_name: "monitoring/windows_static"
+      scrape_interval: 130s
```

Comment:
I'm fine with this, but having a 130s scrape interval may make these metrics useless. Usually we scrape every 5-15s, if you scrape every ~2min and the slowness period you want to observer takes ~30s there is a big chance that you'll miss it. 
That given said, I understand that this is better than nothing, i.e. better to have working metrics with 130s scrape interval than not having the metrics at all.

/lgtm
/approve
/hold
On the other hand, it'd be better if it was a configurable so you could have a short interval for low-pod-density scenarios and increase it for high-pod-density. I think the easiest way to achieve that would be to introduce a param here, with CL2_[1] prefix, e.g. 
```
scrape_interval: {{DefaultParam .CL2_PROMETHEUS_WINDOWS_SCRAPE_INTERVAL 10s}}
```

[1] If a parameter starts with CL2_ you can override it by overriding the corresponding env variable. For manual runs you'd just do `export CL2_MY_PARAM=...`, for prow jobs you can use the ``--env`` flag, see example - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml#L65

Reply:
Agree, and thanks! Made changes using CL2_ prefix. 
Also changed to use `irate()`, as I found `irate()` seems more stable than `rate()`